src/components/Table/Table.story.jsx

```diff
@@ -1263,14 +1264,123 @@ storiesOf('Watson IoT|Table', module)
+        ...i,
+      }))}
+      options={{
+        hasRowSelection: select('hasRowSelection', ['multi', 'single'], 'multi'),
+        hasRowExpansion: false,
+        hasResize: true,
+      }}
+      view={{ table: { selectedIds: array('selectedIds', []) } }}
+    />
+  ))
+  .add(
+    'with resize and initial column widths and hidden column',
+    () => (
+      <FullWidthWrapper>
+        <Table
+          options={{ hasResize: true }}
+          columns={tableColumns.map((i, idx) => ({
+            width: idx % 2 === 0 ? '100px' : '200px',
+            ...i,
+          }))}
+          data={tableData}
+          actions={actions}
+          view={{
+            table: {
+              ordering: defaultOrdering,
+            },
+          }}
+        />
+      </FullWidthWrapper>
+    ),
+    {
+      info: {
+        source: true,
+        propTables: false,
+      },
+    }
+  )
+  .add(
+    'with resize and initial column widths',
+    () => (
+      <Table
+        options={{ hasResize: true }}
+        columns={tableColumns.map((i, idx) => ({
+          width: idx % 2 === 0 ? '100px' : '100px',
+          ...i,
+        }))}
+        data={tableData}
+        actions={actions}
+      />
+    ),
+    {
+      info: {
+        source: true,
+        propTables: false,
+      },
+    }
+  )
+  .add(
+    'with resize and no initial column width',
+    () => (
+      <Table
+        options={{ hasResize: true }}
+        columns={tableColumns}
+        data={tableData}
+        actions={actions}
+      />
+    ),
+    {
+      info: {
+        source: true,
+        propTables: false,
+      },
+    }
+  )
+  .add(
+    'with resize and no initial column width and auto adjusted column widths',
+    () => (
+      <React.Fragment>
+        <p>
+          <strong>Note!</strong> <br />
+          For this configuration to work, the table must be wrapped in a container that has a with
+          defined in other than %. <br />
+          E.g. the FullWidthWrapper used by the storybook examples.
+        </p>
+        <FullWidthWrapper>
+          <Table
+            options={{ hasResize: true, useAutoTableLayoutForResize: true }}
+            columns={tableColumns}
+            data={tableData}
+            actions={actions}
+          />
+        </FullWidthWrapper>
+      </React.Fragment>
+    ),
+    {
+      info: {
+        source: true,
+        propTables: false,
+      },
+    }
+  )
   .add(
-    'with fixed column width',
+    'with fixed column width and no resize',
     () => (
       // You don't need to use styled components, just pass a className to the Table component and use selectors to find the correct column
       <FullWidthWrapper>
         <Table
+          options={{ hasResize: false, hasColumnSelection: true }}
           columns={tableColumns.map((i, idx) => ({
-            width: idx % 2 === 0 ? '20rem' : '10rem',
+            width: idx % 2 === 0 ? '100px' : '100px',
```

Comment:
is there a reason why we changed from rems to px

Reply:
No, for this table without `hasResize: true` it was just a leftover from my testing. I can change back if you want?

There are some restrictions on resizable tables and the unit of initial widths though, see PR description for more info.

Reply:
Changed it back now